language-api/src/main/java/de/jplag/options/LanguageOptionImpl.java

```diff
@@ -0,0 +1,58 @@
+package de.jplag.options;
+
+/**
+ * Default implementation for {@link LanguageOption}
+ * @param <T> The type of the option
+ */
+public class LanguageOptionImpl<T> implements LanguageOption<T> {
```

Comment:
Why not DefaultLanguageOption or something else? I dont like the impl abbreviation, as it is not a very concrete name with that suffic?